src/executables/master_heartbeat.c

```diff
@@ -970,7 +986,14 @@ hb_cluster_job_check_ping (HB_JOB_ARG * arg)
     {
       for (ping_host = hb_Cluster->ping_hosts; ping_host; ping_host = ping_host->next)
 	{
-	  ping_result = hb_check_ping (ping_host->host_name);
+	  if (ping_host->port == 0)	// prm_get_string_value (PRM_ID_HA_PING_HOSTS) != NULL
```

Comment:
```suggestion
	  if (prm_get_string_value (PRM_ID_HA_PING_HOSTS))
```

I think it is more readable

Reply:
Also, this is for consistency with other code such as 1241L in `hb_cluster_job_demote ()`